src/homeblks/volume/tests/vol_gtest.cpp

```diff
@@ -2066,6 +2097,199 @@ class IOTestJob : public TestJob {
     }
 };
 
+// This test job is used to test the IOs with manual requests. For sake of simplicity, we will use the same volume for
+// all requests. The caller needs to load the requests before starting the job. The requests are loaded in the form of
+// Write with three or four parameters and Read with three parameters. The value is optional and is used only for write
+// requests.
+class IOManualTestJob : public TestJob {
```

Comment:
Is there a reason why the existing `IOTestJob::same_lbas()` can't serve the purpose of this basic test?